src/io.jl

```diff
@@ -99,8 +99,14 @@ function Base.write(io::IO, d::MRCData; compress = :none)
     T = datatype(h)
     data = parent(d)
     fswap = bswapfromh(h.machst)
-    for i in eachindex(data)
-        @inbounds sz += write(newio, fswap(T(data[i])))
+    unit_vsize = 4096 ÷ sizeof(T)
```

Comment:
It would be nice if the cache size (either 4096 or `unit_vsize`) was a keyword argument that a user could select. This would also be handy for testing.

Reply:
Thank you for your feedback! It's good idea to set cache size keyword and it would be better for maintenence.